static/app/views/organizationGroupDetails/groupActivityItem.tsx

```diff
@@ -140,6 +141,35 @@ function GroupActivityItem({activity, orgSlug, projectId, author}: Props) {
               author,
             });
       case GroupActivityType.SET_RESOLVED_IN_COMMIT:
+        const deployed_releases: Array<BaseRelease> = [];
+        for (const release of activity.data.commit.releases) {
+          if (release.dateReleased !== null) {
+            deployed_releases.push(release);
+          }
+        }
```

Comment:
Could consider using a filter() instead of for loops:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
since we only use the first one we can probably use .find